clientcontroller/babylon.go

```diff
@@ -510,6 +510,66 @@ func (bc *BabylonController) QueryFinalityProviders() ([]*btcstakingtypes.Finali
 	return fps, nil
 }
 
+func (bc *BabylonController) QueryFinalityProvider(fpPk *btcec.PublicKey) (*btcstakingtypes.QueryFinalityProviderResponse, error) {
+	fpPubKey := bbntypes.NewBIP340PubKeyFromBTCPK(fpPk)
+	res, err := bc.bbnClient.QueryClient.FinalityProvider(fpPubKey.MarshalHex())
+	if err != nil {
+		return nil, fmt.Errorf("failed to query the finality provider %s: %v", fpPubKey.MarshalHex(), err)
+	}
+
+	return res, nil
+}
+
+func (bc *BabylonController) EditFinalityProviderDescription(fpPk *btcec.PublicKey,
```

Comment:
It's better to have a docstring to explain that default value from Babylon will be used if any descriptive fields are not updated